tests/examples/example4-kicad.test.tsx

```diff
@@ -46,6 +47,11 @@ it("example 4: kicad theme demo", async () => {
       />
     </board>,
   )
+  const fs = require('node:fs');
+  const circuitJson = project.getCircuitJson();
+  fs.writeFileSync('circuit.json', JSON.stringify(circuitJson, null, 2)); 
 
   project.render()
+
+//   expect(project).toMatchSchematicSnapshot(import.meta.path)
```

Comment:
Curious why we dont keep the snapshot

Reply:
Cause it's a circle, the new `circuit-to-svg` needs to be published for this snapshot to work

Reply:
And for that to work, I need to publish this `core` 😆 

@seveibar